std/array.d

```diff
@@ -1364,13 +1364,46 @@ if (isSomeString!(T[]) && allSatisfy!(isCharOrStringOrDcharRange, U))
 }
 
 ///
-@safe pure unittest
+@system unittest
```

Comment:
Before this addition, the unittest tested that the code is both @safe and pure. With this addition the first paragraph no longer tests that. Ideally, the added tests should be pure and @safe; if that is not possible, then just create a new unittest block that is @system. (from looking at the code, I don't see why purity is lost; also, the class example should be @safe)